src/js/tourFinder.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * The method that get the user Geolocation, if the user open the Tour Finder page.
+ */
+
+$(document).ready(function() {
+    if (window.location.pathname == '/tour-finder.html') {
+        if ('geolocation' in navigator) {
+            //check geolocation available
+            //try to get user current location using getCurrentPosition() method
+            navigator.geolocation.getCurrentPosition(function(position) {
+                getToursIDs(position.coords);
+            });
+        } else {
+            alert("Browser doesn't support geolocation!");
+        }
+    }
+});
+
+/**
+ * The method that get the tours IDs.
+ */
+
+getToursIDs = coords => {
+    fetch(
+        `https://www.outdooractive.com/api/project/api-dev-oa/nearby/tour/jsonp?location=${coords.latitude},${coords.longitude}&radius=5000000`
+    )
+        .then(response => response.json())
+        .then(data => {
+            console.log(data);
+            if (data) {
+                getTours(data.result);
+                console.log('getToursIDs ', data);
+            } else {
+                console.log("can't take data");
```

Comment:
what is the purpose of this else, try to handle error case (see an example here: https://www.tjvantoll.com/2015/09/13/fetch-and-errors/)